Assets/Plugins/Windows/Core/WindowsConfig.cs

```diff
@@ -25,7 +25,6 @@ namespace PlayEveryWare.EpicOnlineServices
     using System;
 
     // Flags specifically for Windows
-    [Serializable]
```

Comment:
It does not - does the attribute still exist on the base class? The attribute is used mostly for Unity's `JsonUtility`, but since we have supplanted that with Newtonsoft, it's no longer needed, and keeping them in could lead to some confusion in maintaining the project over time.

Reply:
It in fact did - so I also removed it from `PlatformConfig`, and `Config`, and a handful of others. Good catch.